src/app/drive/components/ShareInviteDialog/ShareInviteDialog.tsx

```diff
@@ -0,0 +1,232 @@
+      isValidEmail(email) || usersToInvite.length > 0
+        ? setIsInviteButtonDisabled(false)
+        : setIsInviteButtonDisabled(true);
+    }
+  }, [email]);
+
+  const onAddInviteUser: SubmitHandler<IFormValues> = (formData, event) => {
+    event?.preventDefault();
+    const userInvitedEmail = formData.email;
+    const userInvitedRole = formData.userRole;
+    const userInvited = { email: userInvitedEmail, userRole: userInvitedRole };
+    const isDuplicated = usersToInvite.find((user) => user.email === userInvited.email);
+
+    if (!isDuplicated) {
+      const unique: Array<UsersToInvite> = [...usersToInvite];
+      unique.push(userInvited);
+      setUsersToInvite(unique);
+    }
+    resetField('email');
+  };
+
+  const onEditRole = () => {
+    //   Edit user role
+  };
+
+  const onInvite = () => {
+    // Ivite added users
+  };
+
+  return (
+    <div>
+      <div className="flex h-16 w-full items-center justify-between space-x-5 border-b border-gray-10 px-5">
+        <h4
+          className="flex max-w-full items-center overflow-hidden overflow-ellipsis whitespace-nowrap text-xl font-medium"
+          title={translate('modals.shareModal.invite.title')}
+        >
+          <ArrowLeft onClick={props.onInviteUser} size={22} className="mr-2 cursor-pointer" />
+          {translate('modals.shareModal.invite.title')}
+        </h4>
+      </div>
+
+      <div className="p-5">
+        <form className="m flex w-full" onSubmit={handleSubmit(onAddInviteUser)}>
+          <input
+            className="no-ring semi-dense mr-2 w-full flex-grow border border-neutral-30"
+            placeholder={translate('form.fields.email.placeholder') as string}
+            type="text"
+            {...register('email', {
+              required: true,
+              pattern: emailRegex,
+            })}
+          />
+          <Listbox value={userRole} onChange={setUserRole}>
+            {({ open }) => (
+              <div className="relative">
+                <Listbox.Button value={userRole} {...register('userRole')} name="userRole">
+                  <Button variant="secondary">
+                    <span className="capitalize">{userRole}</span>
+                    <CaretDown
+                      size={24}
+                      className={`${open ? 'rotate-180 transform transition' : 'rotate-0 transform transition'}`}
+                    />
+                  </Button>
+                </Listbox.Button>
+                <Listbox.Options className="absolute right-0 z-10 mt-1 w-40 transform whitespace-nowrap rounded-lg border border-gray-10 bg-white p-1 shadow-subtle transition-all duration-50 ease-out">
+                  <Listbox.Option
+                    key="editor"
+                    value="editor"
+                    className="flex h-9 w-full cursor-pointer items-center justify-start justify-between space-x-3 rounded-lg py-2 px-3 px-3 text-base font-medium hover:bg-gray-5"
```

Comment:
Every Listbox.Options it has two justify properties: `justify-start justify-between` and `px-3` is duplicated

Reply:
Fixed